### PR TITLE
[Feat]Card

### PR DESCRIPTION
**Pull request by DeepakkPatil**
Added Card component and  its css file

### CODE REVIEW

File: components/Card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./Card.module.scss";
```
Comment (shishirtiwari23) [inline components/Card/Card.tsx:2]:
```jsx
//import like this 
import styles from "./fileName.moudle.scss"

//And use it like this 
<div className={styles.yourClassName}></div>
```

File: components/Card/Card.module.scss [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  height: 300px;
+  border: 1px solid black;
+  display: flex;
+  justify-content: first baseline;
+  border-radius: 8px;
+  flex-direction: column;
+  box-shadow: 2px 5px #888888;
+}
+
+.Card-header{
```
Comment (shishirtiwari23) [inline components/Card/Card.module.scss:13]:
```scss
//use camel case notation
.cardHeader{
//your CSS here
}
//also try to do a little nesting (not much)
```

**Review by shishirtiwari23 — CHANGES_REQUESTED** [review]
Please make these changes

**Review by DeepakkPatil — COMMENTED** [review]
Changes Done

**Review by shishirtiwari23 — APPROVED** [review]
Looks good for now

**Pull request merged**